ios/FluentUI/Label/BadgeLabel.swift

```diff
@@ -29,6 +30,16 @@ class BadgeLabel: UILabel {
         isHidden = true
     }
 
+    override func didMoveToWindow() {
+        super.didMoveToWindow()
+
+        guard let window = window, shouldUseWindowColor else {
```

Comment:
nit: can you put the shouldUseWindowColor first? then it doesn't need to access the window property if it's not needed.

Reply:
Done!!